File: easy/rna-transcription.js
export const toRna = (string) => {
  return [...string]
    .map((char) => {
      switch (char) {
        case "G":
          return "C";
          break;
        case "C":
          return "G";
          break;
        case "T":
          return "A";
          break;
        case "A":
          return "U";
          break;
      }
    })
    .join("");
};

File: python/rna_transcription.py
def to_rna(dna_strand):
    return_string = ""
    for letter in dna_strand:
        if letter == "G":
            return_string += "C"
        elif letter == "C":
            return_string += "G"
        elif letter == "T":
            return_string += "A"
        elif letter == "A":
            return_string += "U"
    return return_string
